easybuild/easyconfigs/s/snap/snap-2013-11-29-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,40 @@
+#
+# Author: Fenglai Liu
+# fenglai@accre.vanderbilt.edu
+# Vanderbilt University
+#
+easyblock = 'MakeCp'
+
+name = 'snap'
```

Comment:
@murfreesboro Should be `SNAP` (see description)?

Reply:
This is not SNAP. They have the same name, but the program actually is different.

Reply:
In the easybuild repo we already have a program named "SNAP", but that one is a different program.... in this reason I named it as "snap".

Reply:
@murfreesboro I don't think we should artificially mangle the software name to try and dance around a name clash...

In this case the version numbering is different enough that this can serve as a discriminator, so I would still stick to `SNAP` since that seems to be the official software name.

If you change the name to `SNAP`, please rename the patch file as well accordingly (and change `sources` to `SOURCELOWER_TAR_GZ`)